src/NoFrixion.MoneyMoov/Models/PaymentRequests/PaymentRequestCustomField.cs

```diff
@@ -23,5 +23,5 @@ public class PaymentRequestCustomField
     
     public bool DisplayToPayer { get; set; }
     
-    public int Priority { get; set; }
+    public int SequenceNumber { get; set; }
```

Comment:
Maybe DisplayOrder would be better? It's not really a sequence number unless you're going to enforce some sort of sequence.